src/org/jitsi/impl/neomedia/MediaStreamImpl.java

```diff
@@ -1495,19 +1495,16 @@ public MediaDirection getDirection()
      */
     public byte getDynamicRTPPayloadType(String encoding)
     {
-        synchronized (dynamicRTPPayloadTypes)
+        for (Map.Entry<Byte, MediaFormat> dynamicRTPPayloadType
```

Comment:
I'm sorry, I misread the code. There's an intermediate call that actually copies the set.